docs/throttling.md

```diff
@@ -1,2 +1,22 @@
-## Throttling
+### Background
+
+We have had to disable network throttling in Synthetics for browser based monitors. This is intended to be a temporary measure until we can release an update in the future to enable it again.
```

Comment:
```suggestion
We have disabled network throttling in Synthetics for browser-based monitors. This is a temporary measure and throttling will become available in a future update.
```
Slightly simplified. "Future" is quite vague and could sound like 'next year'. If we have an approximate idea, we could reassure users by for example saying "will become available in the coming weeks".

Reply:
Taken your suggestion (we don't have an estimate of time unfortunately so will have to leave this vague)